pkg/providers/instance/instance.go

```diff
@@ -138,36 +145,39 @@ func (p *DefaultProvider) Get(ctx context.Context, id string) (*Instance, error)
 
 func (p *DefaultProvider) List(ctx context.Context) ([]*Instance, error) {
 	var out = &ec2.DescribeInstancesOutput{}
-	err := p.ec2api.DescribeInstancesPagesWithContext(ctx, &ec2.DescribeInstancesInput{
-		Filters: []*ec2.Filter{
+
+	paginator := ec2.NewDescribeInstancesPaginator(p.ec2api, lo.ToPtr(ec2.DescribeInstancesInput{
```

Comment:
```suggestion
	paginator := ec2.NewDescribeInstancesPaginator(p.ec2api, &ec2.DescribeInstancesInput{
```

Why not just use an ampersand like we used to do and get a reference to the object directly rather than using `lo.ToPtr`?